### PR TITLE
Adding cleanat script to clean the TLOG audit tables

### PR DESCRIPTION
**Pull request by sbroeder**
from the instance.

This is in response to JIRA 1644

### CODE REVIEW

**Comment by sbroeder** [comment]
[TRAFODION-1644] adding a cleanat script to allow removal of TRAFODION._DTM_* tables

**Comment by svarnau** [comment]
jenkins, add user

File: core/sqf/sql/scripts/cleanat [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sqstop
```
Comment (DaveBirdsall) [inline core/sqf/sql/scripts/cleanat]: Needs an Apache copyright.

**Comment by sbroeder** [comment]
Of course!  Thanks Dave

**Pull request merged**